main.go

```diff
@@ -210,15 +214,29 @@ func addMetrics(ctx context.Context) {
 	}
 }
 
-func getWatchNamespace() (string, error) {
-	// WatchNamespaceEnvVar is the constant for env variable WATCH_NAMESPACE
-	// which specifies the Namespace to watch.
+// This function could accept a string which is the object name.
+// That could then be added to the loginfo messages.
```

Comment:
These comments I left to illustrate that an object name could be passed which could then be used as part of the logging information.  Just an idea.